cli/crates/typed-resolvers/tests/schema_types/keyword_names.expected.ts

```diff
@@ -12,7 +12,7 @@
 //  }
 
 export type Schema = {
-  'holycow': | Schema['type'] | Schema['interface'] | Schema['object'] | Schema['union'];
+  'holycow': | Schema['type'] | Schema['interface'] | Schema['object'];
```

Comment:
This line is not obvious because of the extremely confusing naming in this test, but the change is correct: `union` here is an interface (I know). The [Good Spec says interfaces can't be union members](http://spec.graphql.org/October2021/#sec-Unions.Type-Validation) so not taking them into account here does not matter because it won't happen in a valid GraphQL schema.